tmt/base.py

```diff
@@ -1917,6 +1917,12 @@ def init(path, template, force, **kwargs):
             if path == tree.root:
                 echo("Tree '{}' already exists.".format(tree.root))
             else:
+                # Are we creating a nested tree?
+                if path > tree.root:
+                    echo("Path '{}' already has a parent root '{}'.".format(path, tree.root))
+                    answer = input("Do you really want to initialize a nested tree? (Yes/No): ")
```

Comment:
I'd prefer to use https://click.palletsprojects.com/en/8.1.x/api/#click.confirm

@psss Could 'force' be used to "don't ask, just do it" which will skip the prompt? Cleaner would be to add new option but is it worth that?

Reply:
`input` is updated to use `click.confirm`. If option `--force` is required, please let me know, I'll add it accordingly.